ui/pages/accept-invite/index.js

```diff
@@ -18,6 +21,32 @@ export default function AcceptInvite() {
 
   return (
     <div className='flex w-full flex-col items-center px-10 pt-4 pb-6'>
+      <h1 className='text-base font-bold leading-snug'>Welcome to Infra</h1>
```

Comment:
Change to: `Create your account` ?

Reply:
Went with `Accept invitation`. The screenshot it updated with the new format.